### PR TITLE
Update eventhub to enable live testing in sovereign clouds for multiple services

### PR DESCRIPTION
**Pull request by zedy-wj**
These changes enable eventhub to run live tests against AzureUsGovernment and AzureChina clouds. The sovereign cloud live tests will be green with the changes.
@benbp @danieljurek @jongio

### CODE REVIEW

**Comment by zedy-wj** [comment]
@benbp @danieljurek @jongio

File: sdk/eventhub/tests.data.yml [hunk 1]
```diff
@@ -6,5 +6,5 @@ extends:
     MaxParallel: 6
     ServiceDirectory: eventhub
     SDKType: data
-    TimeoutInMinutes: 190
-    Clouds: 'Public,Canary'
\ No newline at end of file
+    TimeoutInMinutes: 240
+    Clouds: 'Public,Preview,UsGov,China,Canary'
```
Comment (jsquire) [inline sdk/eventhub/tests.data.yml:10]: Do we want to enable the track one tests in sovereign clouds?   I think we'll want @serkantkaraca weigh in.

File: sdk/eventhub/tests.yml [hunk 2]
```diff
@@ -6,5 +6,5 @@ extends:
     MaxParallel: 6
     ServiceDirectory: eventhub
     SDKType: client
-    TimeoutInMinutes: 190
-    Clouds: 'Public,Canary'
\ No newline at end of file
+    TimeoutInMinutes: 240
+     Clouds: 'Public,Preview,UsGov,China,Canary'
```
Comment (jsquire) [inline sdk/eventhub/tests.yml:10]:
```suggestion
    Clouds: 'Public,Preview,UsGov,China,Canary'
```

File: sdk/eventhub/tests.data.yml (hunk 1 above)
Comment (benbp) [reply sdk/eventhub/tests.data.yml:10]: I think we can remove `Preview` regardless.

Comment (zedy-wj) [reply sdk/eventhub/tests.data.yml]: OK, l have removed `Preview` .

**Review by jsquire — APPROVED** [review]

**Comment by benbp** [comment]
Thanks @zedy-wj!

**Pull request merged**